docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -49,6 +49,7 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | [UnixTimestampOf](../function_datetime/#unixtimestampof) | [`BIGINT`](../type_int) | ([`TIMEUUID`](../type_uuid)) | Conversion |
 | [UUID](../function_datetime/#uuid) | [`UUID`](../type_uuid) | () | Returns a version 4 UUID |
 | WriteTime | [`BIGINT`](../type_int) | (<AnyType>) | Returns the time when the column was written |
+| [partition_hash](#partition-hash-function) | [`BIGINT`](../type_int) | () | Returns a `uint16` of partition keys |
```

Comment:
What does returns a `unit16` of partition keys mean?

This description is incorrect.